provider/test/utils/hooks.ts

```diff
@@ -51,7 +51,7 @@ export async function mochaGlobalSetup(this: GlobalFixturesContext) {
   }
 
   this.testApp = httpServer.createServer({ root: testAppDir });
-  this.testApp.listen(8081, '0.0.0.0', function () {});
+  this.testApp.listen(8081, function () {});
```

Comment:
Is this empty callback needed at all? Can it be removed?

Reply:
Yes, it's unnecessary.